agnocomplete/widgets.py

```diff
@@ -79,6 +78,34 @@ def build_attrs(self, base_attrs, extra_attrs=None):
                 base_attrs, extra_attrs)
             return self._agnocomplete_build_attrs(attrs)
 
+        """
+        Returns the selected option set in order to retrieve the behaviour of
+        AgnocompleteWidgetMixin.render_options()
+        """
+        def _agnocomplete_selected_options(self, options, value):
+            selected_options = {}
+
+            for opt in options:
+                opt_value = text(opt.get('value'))
+                opt_selected = (opt_value in value)
+
+                opt['selected'] = opt_selected
+                opt['attrs'][u'selected'] = opt_selected
```

Comment:
Nitpick: if we put `from __future__ import unicode_literals` at the top of the file, do you think we can get a ride of `u'selected' here?

Reply:
I agree.